cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectClusterTest.java

```diff
@@ -29,21 +29,25 @@
     private final String configurationJson = "{\"foo\":\"bar\"}";
     private final String expectedConfiguration = "group.id=connect-cluster\n" +
             "key.converter=org.apache.kafka.connect.json.JsonConverter\n" +
+            "internal.key.converter.schemas.enable=false\n" +
             "value.converter=org.apache.kafka.connect.json.JsonConverter\n" +
             "config.storage.topic=connect-cluster-configs\n" +
             "status.storage.topic=connect-cluster-status\n" +
             "offset.storage.topic=connect-cluster-offsets\n" +
             "foo=bar\n" +
             "internal.key.converter=org.apache.kafka.connect.json.JsonConverter\n" +
+            "internal.value.converter.schemas.enable=false\n" +
             "internal.value.converter=org.apache.kafka.connect.json.JsonConverter\n";
-    private final String defaultConfiguration = "internal.value.converter=org.apache.kafka.connect.json.JsonConverter\n" +
-            "offset.storage.topic=connect-cluster-offsets\n" +
-            "group.id=connect-cluster\n" +
+    private final String defaultConfiguration = "group.id=connect-cluster\n" +
```

Comment:
It's a trival comment, but you seems to have reordered these configs info a *more* random order than they were before.